doc/manual.org

```diff
@@ -1175,9 +1175,11 @@ You should set the =build:= field of your =<package>.opam= file as
 follows:
 
 #+begin_src
-build: [["jbuilder" "build" "--only-packages" "<package>" "--root" "." "-j" jobs "@install"]]
+build: [["jbuilder" "build" "--release" "<package>" "-j" jobs]]
```

Comment:
I answer myself, now @install is the default (https://github.com/janestreet/jbuilder/pull/47). Cool!